java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -31,6 +35,13 @@ class DevToolsService : Service() {
     private val scope = CoroutineScope(coroutineContext)
     private lateinit var binder: DevToolsBinder
     private val devToolsServer = DevWebServerImpl
+    private val connectionFactory = DevToolsConnectionFactory(
+        this@DevToolsService,
+        StorageService::class.java
```

Comment:
How will this get overridden by clients to reference a storage service subclass instead?

Reply:
Changed to the getStorageClass() method. I tested it using the system health test app (which subclasses storage service), and saw the storage keys on the client side.